src/main/io/rc_controls.c

```diff
@@ -392,260 +400,126 @@ void applyStepAdjustment(controlRateConfig_t *controlRateConfig, uint8_t adjustm
     }
     switch(adjustmentFunction) {
```

Comment:
`offsetof()` can help with address part ...
dual adjustments are a bit harder ... maybe add new type (ADJUSTMENT_MODE_MULTI) to adjustmentConfig_t and pass individual adjustments indexes in data part ...
Also applyStepAdjustment table may be sparse to save unused functions ..